src/core/auth/qgsauthmanager.h

```diff
@@ -73,6 +74,8 @@ class CORE_EXPORT QgsAuthManager : public QObject
 
   public:
 
+    static const QgsSettingsEntryBool *settingsGenerateRandomPasswordForPasswordHelper SIP_SKIP;
```

Comment:
comment please.

Reply:
No other settings entries are commented -- they are self documenting, in the definition